pom.xml

```diff
@@ -147,16 +147,16 @@
     <module>bom</module>
     <module>broker</module>
     <module>core</module>
+    <module>service-base</module>
```

Comment:
You're right, I wasn't thinking properly. I need to add proper dependencies to the example module. I assume you want to keep modules in alphabetical order?